neo/Network/P2P/Payloads/VersionServices.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Neo.Network.P2P.Payloads
+{
+    [Flags]
+    public enum VersionServices : ulong
```

Comment:
In the proposal it mentions the SSL, to the version service.

Reply:
I think that is better in a flag

Reply:
Use `VersionServices` or add a new field `SupportedProtocols`?